[2025-03-23T18:54:46.828+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-23T18:54:46.857+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: schedule.move_cloud_clientes_produtos manual__2025-03-23T18:54:43.577498+00:00 [queued]>
[2025-03-23T18:54:46.870+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: schedule.move_cloud_clientes_produtos manual__2025-03-23T18:54:43.577498+00:00 [queued]>
[2025-03-23T18:54:46.870+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-23T18:54:46.892+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): move_cloud_clientes_produtos> on 2025-03-23 18:54:43.577498+00:00
[2025-03-23T18:54:46.897+0000] {standard_task_runner.py:72} INFO - Started process 1152 to run task
[2025-03-23T18:54:46.908+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'schedule', 'move_cloud_clientes_produtos', 'manual__2025-03-23T18:54:43.577498+00:00', '--job-id', '1895', '--raw', '--subdir', 'DAGS_FOLDER/dag_master.py', '--cfg-path', '/tmp/tmpckb0xh4z']
[2025-03-23T18:54:46.910+0000] {standard_task_runner.py:105} INFO - Job 1895: Subtask move_cloud_clientes_produtos
[2025-03-23T18:54:46.935+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.8/site-packages/airflow/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-23T18:54:46.995+0000] {task_command.py:467} INFO - Running <TaskInstance: schedule.move_cloud_clientes_produtos manual__2025-03-23T18:54:43.577498+00:00 [running]> on host b612e1a2c49c
[2025-03-23T18:54:47.109+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='schedule' AIRFLOW_CTX_TASK_ID='move_cloud_clientes_produtos' AIRFLOW_CTX_EXECUTION_DATE='2025-03-23T18:54:43.577498+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-23T18:54:43.577498+00:00'
[2025-03-23T18:54:47.111+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-23T18:54:47.131+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-23T18:54:47.135+0000] {subprocess.py:88} INFO - Running command: ['/bin/bash', '-c', 'docker exec linux_python /bin/bash -c \'/workspace/scripts/run_move_file.sh /path/to/move_cloud/clientes_produtos/clientes_produtos.csv table_ingestion_files/clientesxprod/20250323/clientes_produtos_1742756086.csv && echo "Script Executado com Sucesso!"\'']
[2025-03-23T18:54:47.151+0000] {subprocess.py:99} INFO - Output:
[2025-03-23T18:54:47.786+0000] {subprocess.py:106} INFO - 2025-03-23 18:54:47,781 - INFO - Iniciando o upload do arquivo '/path/to/move_cloud/clientes_produtos/clientes_produtos.csv' para o blob 'table_ingestion_files/clientesxprod/20250323/clientes_produtos_1742756086.csv' no container 'filedataimport'.
[2025-03-23T18:54:47.792+0000] {subprocess.py:106} INFO - 2025-03-23 18:54:47,781 - INFO - Blob URL gerada: https://stagedatamaster.blob.core.windows.net/filedataimport/table_ingestion_files/clientesxprod/20250323/clientes_produtos_1742756086.csv?sp=cw&st=2024-10-20T02:42:12Z&se=2026-10-20T10:42:12Z&spr=https&sv=2022-11-02&sr=c&sig=MaoRwZWwtKEx5vDafwkbeHkx1Rir6bWRYy0FB3OOs58%3D
[2025-03-23T18:54:47.794+0000] {subprocess.py:106} INFO - 2025-03-23 18:54:47,787 - INFO - Enviando o arquivo '/path/to/move_cloud/clientes_produtos/clientes_produtos.csv' para o Azure Blob Storage.
[2025-03-23T18:54:47.795+0000] {subprocess.py:106} INFO - 2025-03-23 18:54:47,789 - INFO - Request URL: 'https://stagedatamaster.blob.core.windows.net/filedataimport/table_ingestion_files/clientesxprod/20250323/clientes_produtos_1742756086.csv?sp=REDACTED&st=REDACTED&se=REDACTED&spr=REDACTED&sv=REDACTED&sr=REDACTED&sig=REDACTED'
[2025-03-23T18:54:47.797+0000] {subprocess.py:106} INFO - Request method: 'PUT'
[2025-03-23T18:54:47.797+0000] {subprocess.py:106} INFO - Request headers:
[2025-03-23T18:54:47.798+0000] {subprocess.py:106} INFO -     'Content-Length': '157128'
[2025-03-23T18:54:47.799+0000] {subprocess.py:106} INFO -     'x-ms-blob-type': 'REDACTED'
[2025-03-23T18:54:47.799+0000] {subprocess.py:106} INFO -     'If-None-Match': '*'
[2025-03-23T18:54:47.799+0000] {subprocess.py:106} INFO -     'x-ms-version': 'REDACTED'
[2025-03-23T18:54:47.799+0000] {subprocess.py:106} INFO -     'Content-Type': 'application/octet-stream'
[2025-03-23T18:54:47.800+0000] {subprocess.py:106} INFO -     'Accept': 'application/xml'
[2025-03-23T18:54:47.800+0000] {subprocess.py:106} INFO -     'User-Agent': 'azsdk-python-storage-blob/12.24.1 Python/3.12.9 (Linux-6.10.14-linuxkit-aarch64-with-glibc2.36)'
[2025-03-23T18:54:47.800+0000] {subprocess.py:106} INFO -     'x-ms-date': 'REDACTED'
[2025-03-23T18:54:47.801+0000] {subprocess.py:106} INFO -     'x-ms-client-request-id': '4aeec5b2-0818-11f0-bc3e-0242ac120004'
[2025-03-23T18:54:47.801+0000] {subprocess.py:106} INFO - A body is sent with the request
[2025-03-23T18:54:48.025+0000] {subprocess.py:106} INFO - 2025-03-23 18:54:48,023 - INFO - Response status: 409
[2025-03-23T18:54:48.027+0000] {subprocess.py:106} INFO - Response headers:
[2025-03-23T18:54:48.028+0000] {subprocess.py:106} INFO -     'Content-Length': '220'
[2025-03-23T18:54:48.028+0000] {subprocess.py:106} INFO -     'Content-Type': 'application/xml'
[2025-03-23T18:54:48.028+0000] {subprocess.py:106} INFO -     'Server': 'Windows-Azure-Blob/1.0 Microsoft-HTTPAPI/2.0'
[2025-03-23T18:54:48.029+0000] {subprocess.py:106} INFO -     'x-ms-request-id': '5beadaf1-401e-0031-1225-9c0f9d000000'
[2025-03-23T18:54:48.029+0000] {subprocess.py:106} INFO -     'x-ms-client-request-id': '4aeec5b2-0818-11f0-bc3e-0242ac120004'
[2025-03-23T18:54:48.030+0000] {subprocess.py:106} INFO -     'x-ms-version': 'REDACTED'
[2025-03-23T18:54:48.030+0000] {subprocess.py:106} INFO -     'x-ms-error-code': 'BlobAlreadyExists'
[2025-03-23T18:54:48.030+0000] {subprocess.py:106} INFO -     'Date': 'Sun, 23 Mar 2025 18:54:47 GMT'
[2025-03-23T18:54:48.030+0000] {subprocess.py:106} INFO - 2025-03-23 18:54:48,024 - ERROR - Erro ao enviar o arquivo '/path/to/move_cloud/clientes_produtos/clientes_produtos.csv': The specified blob already exists.
[2025-03-23T18:54:48.031+0000] {subprocess.py:106} INFO - RequestId:5beadaf1-401e-0031-1225-9c0f9d000000
[2025-03-23T18:54:48.031+0000] {subprocess.py:106} INFO - Time:2025-03-23T18:54:47.9513094Z
[2025-03-23T18:54:48.031+0000] {subprocess.py:106} INFO - ErrorCode:BlobAlreadyExists
[2025-03-23T18:54:48.032+0000] {subprocess.py:106} INFO - Content: <?xml version="1.0" encoding="utf-8"?><Error><Code>BlobAlreadyExists</Code><Message>The specified blob already exists.
[2025-03-23T18:54:48.032+0000] {subprocess.py:106} INFO - RequestId:5beadaf1-401e-0031-1225-9c0f9d000000
[2025-03-23T18:54:48.032+0000] {subprocess.py:106} INFO - Time:2025-03-23T18:54:47.9513094Z</Message></Error>
[2025-03-23T18:54:48.033+0000] {subprocess.py:106} INFO - 2025-03-23 18:54:48,024 - ERROR - Falha no processo de upload: The specified blob already exists.
[2025-03-23T18:54:48.033+0000] {subprocess.py:106} INFO - RequestId:5beadaf1-401e-0031-1225-9c0f9d000000
[2025-03-23T18:54:48.033+0000] {subprocess.py:106} INFO - Time:2025-03-23T18:54:47.9513094Z
[2025-03-23T18:54:48.033+0000] {subprocess.py:106} INFO - ErrorCode:BlobAlreadyExists
[2025-03-23T18:54:48.034+0000] {subprocess.py:106} INFO - Content: <?xml version="1.0" encoding="utf-8"?><Error><Code>BlobAlreadyExists</Code><Message>The specified blob already exists.
[2025-03-23T18:54:48.034+0000] {subprocess.py:106} INFO - RequestId:5beadaf1-401e-0031-1225-9c0f9d000000
[2025-03-23T18:54:48.035+0000] {subprocess.py:106} INFO - Time:2025-03-23T18:54:47.9513094Z</Message></Error>
[2025-03-23T18:54:48.115+0000] {subprocess.py:106} INFO - Script Executado com Sucesso!
[2025-03-23T18:54:48.133+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-23T18:54:48.206+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-23T18:54:48.207+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=schedule, task_id=move_cloud_clientes_produtos, run_id=manual__2025-03-23T18:54:43.577498+00:00, execution_date=20250323T185443, start_date=20250323T185446, end_date=20250323T185448
[2025-03-23T18:54:48.287+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-23T18:54:48.358+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
